docs/connecting-data/connecting-to-apis-and-external-data/harmonic-api.md

```diff
@@ -1,9 +1,99 @@
 ---
-description: How to Connect DataDistillr to the Harmonic API
+description: How to Connect DataDistillr to the Harmonic.ai API
 ---
 
-# Connecting to Harmonic
+# Connecting to Harmonic.ai
 
-## First Steps with Harmonic
+## First Steps with Harmonic.ai
+Set up an account with [Harmonic.ai](https://console.harmonic.ai/signup){target=_blank}.
 
-Details coming soon!
\ No newline at end of file
+???+ cost "Cost"
+
+    [Request access](https://www.harmonic.ai/request-access){target=_blank} to find out more about Harmonic.ai. 
+
+
+???+ rlimit "Rate Limits"
+
+    No rate limits on currently supported endpoints
+
+## How to Connect DataDistillr to Harmonic.ai
+To set up a data source connection for Harmonic.ai, you will need to have:
+
+- A unique [name](#name) for your data source connection to be used in queries.
+- The [API Key](#api-key) generated by Harmonic.ai.
+- The [WEBSITE DOMAIN](#website-domain) generated by Harmonic.ai.
+
+
+
+### Data Source Form
+
+To locate the Harmonic.ai form, follow the steps in [Connecting Your Data to DataDistillr](../../). When you get to the window to choose the data source type, select API as shown below.&#x20;
+
+![Select API from the available choices][image-5]
+
+On the API screen, select Harmonic.ai from list of API forms as shown in the image below.
+
+![Select Harmonic.ai API from available choices][image-6]
+
+The following form will appear. Instructions can be found below on how to find the information required to fill each field on the Harmonic.ai API form.
+
+Once you have filled out all the fields, press the green 'Save' button, and your API will be connected!
+
+![Harmonic.ai Form][image-1]
+
+### Name
+
+Enter any name that will help you recognize this data source from within your query window. &#x20;
+
+!!! info "Acceptable characters include"
+
+    * lowercase alphanumeric characters
+    * underscores
+
+### API Key
+
+Retrieve your API Key from Harmonic.ai
+
+
+## Endpoints
+
+The table below shows a list of endpoints available to connect to within the DataDistillr application. If you need to connect to any endpoints not listed in the table below, please use the [Custom API](custom-apis.md) Form.
+
+| Endpoint     | Required Params | Optional  Params | Description                              |
+|--------------|-----------------|------------------|------------------------------------------|
+| `/companies` | website_domain  |                  | Returns the list of all Synthetic tests. |
+
+
+### Nav Tree
+
+The endpoint above will display as follows in the nav tree once your API has successfully connected.
+
+![Harmonic.ai Endpoints][image-3]
+
+## Sample Queries
+
+The following queries are intended to help you get started, and make life simpler querying within your API.
+
+For the following examples, suppose that my Harmonic.ai data source was called `myHarmonic.ai` and I want to query an endpoint. The endpoint goes after the Harmonic.ai data source name:
```

Comment:
I believe you mean the data source was called 'myharmonicapi' rather than 'myHarmonic.ai'?